unified-scheduler-pool/src/lib.rs

```diff
@@ -1800,7 +1800,7 @@ impl<S: SpawnableScheduler<TH>, TH: TaskHandler> ThreadManager<S, TH> {
                     // UnpauseOpenedSubchannel won't be used by itself, which would leave
                     // `result_with_timings` uninitialized after sending it via
                     // session_result_sender just above
-                    let mut new_result_with_timings = initialized_result_with_timings();
+                    let mut new_result_with_timings = None;
```

Comment:
By making `Option<_>`-ed, we can codify that the exact sequence of OpenSubchannel => CloseSubchannel.